prerequisite_changes.py

```diff
@@ -176,8 +177,86 @@ def main():
                 "== 4",
                 "== ${length}")
     if service == "core-data":
-        # TBD
-        print("This is core-data service related changes")
+        # Skipping this test
```

Comment:
why skipping the tests? Are we not total same test cases running in actual EdgeX-TAF side?
@bhavikkothari-eaton

Reply:
@rajatkumar-eaton skipped tests are taking count which depends on docker image and it is not in our image. I also added comment in file.